web/static/js/components/category_column.jsx

```diff
@@ -13,7 +13,7 @@ function CategoryColumn(props) {
   const emoticonUnicode = categoryToEmoticonUnicodeMap[props.category]
   const filteredIdeas = props.ideas.filter(idea => idea.category === props.category)
   const filteredIdeasList = filteredIdeas.map(idea =>
-    <li className="item" title={idea.body} key={idea.id}>{idea.body}</li>,
+    <li className="item" title={idea.body} key={idea.id}>{props.currentUser}: {idea.body}</li>,
```

Comment:
hey @caporta, won't this result in _all_ ideas being prefixed with `<CURRENT_USER_NAME>:`, even when they've been submitted by someone else?